.github/CODEOWNERS

```diff
@@ -1,13 +1,13 @@
-crates/core @roeap @rtyler @hntd187 @ion-elgreco
-crates/deltalake @roeap @rtyler @hntd187 @ion-elgreco
+crates/core @roeap @rtyler @hntd187
 crates/aws @rtyler
 crates/lakefs @ion-elgreco
+crates/catalog-glue @rtyler
 crates/catalog-unity @roeap @hntd187
 
 delta-inspect/ @rtyler
 
 proofs/ @houqp
-python/ @wjones127 @roeap @ion-elgreco
```

Comment:
I'm happy to keep receiving pings for python to share the load a bit with @ion-elgreco, even though he knows the python code much better than me by now 😆.